src/components/Description.jsx

```diff
@@ -4,8 +4,8 @@ import {Alert} from '@mui/material';
 const Description = () => {
   return (
     <Alert severity="info" className='app-desc'>
-      "Get NASA's Astronomy Picture of the Day (APOD)
-       in daily text"
+      "Get NASA Astronomy Picture of the Day (APOD)
```

Comment:
Remove the quotes. Unless you want to keep them for some reason?

Reply:
I don't mind

Reply:
Ok.. fair enough.